src/pages/iou/IOUDetailsModal.js

```diff
@@ -123,12 +207,41 @@ class IOUDetailsModal extends Component {
                         {(this.props.iouReport.hasOutstandingIOU
                             && this.props.iouReport.managerEmail === sessionEmail && (
                             <View style={styles.p5}>
-                                <Button
-                                    success
-                                    text={this.props.translate('iou.settleElsewhere')}
-                                    isLoading={this.props.iou.loading}
-                                    onPress={this.performIOUPayment}
-                                />
+                                {this.state.paymentOptions.length > 1 ? (
+                                    <ButtonWithDropdown
+                                        success
+                                        buttonText={paymentTypeText[this.state.paymentType]}
```

Comment:
NAB: Could be DRY'd even further by making a const for this? `paymentTypeText[this.state.paymentType]`